Automattic/newspack-theme
# feat: add Apple News icon for the social links menu

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a social links icon for Apple News.

Closes #1173 

### How to test the changes in this Pull Request:

1. Set up a social links menu; include a link for an Apple News page (eg. https://apple.news/T9iApGZVCQRKp_KmxbhpkZw)
2. View on the front end; note that the Apple News icon is using the fallback 'link' icon:

![image](https://user-images.githubusercontent.com/177561/101699557-2fd10200-3a30-11eb-9b9e-72ce29a34f44.png)

3. Apply the PR.
4. Confirm that the Apple News link now uses the correct icon, and that it seems to be scaled well compared to your other icons:

![image](https://user-images.githubusercontent.com/177561/101699500-1465f700-3a30-11eb-9782-6149da994a48.png)

### Other information:
